### DISCUSSION TITLE
Event Calendar

### DISCUSSION THREAD

**Discussion by vikramthangaraj** [primefaces/community#1566] [PrimeNG] [answered]
Day Month, Week and Weekdays Calendar are much Interested and also Scheduling functionality as in the Outlook Calendar. Is there anyway we can achieve this?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, the calendar component is not as advanced as the Outlook calendar component. PrimeNG Calendar is an input component that allows selecting day/month/year/hour.
